### ISSUE TITLE
PDOException: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'series_id' in field list is ambiguous

### ISSUE THREAD

**Issue by tleilax** [elan-ev/studip-opencast-plugin#239]
```
2020/04/08 13:19:01 [error] 4713#4713: *80993694 FastCGI sent in stderr: "PHP message: PDOException: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'series_id' in field list is ambiguous in /var/www/studip-4.3/lib/classes/StudipPDOStatement.php:154
Stack trace:
#0 /var/www/studip-4.3/lib/classes/StudipPDOStatement.php(154): PDOStatement->execute()
#1 /var/www/studip-4.3/public/plugins_packages/elan-ev/OpenCast/models/OCSeriesModel.php(61): StudipPDOStatement->execute(Array)
#2 /var/www/studip-4.3/public/plugins_packages/elan-ev/OpenCast/controllers/ajax.php(71): OCSeriesModel::getSeriesForUser('8208aefdcca7677...')
#3 /var/www/studip-4.3/vendor/trails/trails.php(557): AjaxController->getseries_action()
#4 /var/www/studip-4.3/app/controllers/studip_controller.php(117): Trails_Controller->perform('getseries')
#5 /var/www/studip-4.3/vendor/trails/trails.php(154): StudipController->perform('getseries')
#6 /var/www/studip-4.3/vendor/trails/trails.php(121): Trails_Dispatcher->map_uri_to_response('ajax/getseries')
#7 /var/www/studip-4.3/lib/plugins/core/StudIPPlugin.clas" while reading response header from upstream, client: 10.0.41.29, server: elearning.uni-oldenburg.de, request: "GET /plugins.php/opencast/ajax/getseries HTTP/2.0", upstream: "fastcgi://127.0.0.1:9000", host: "elearning.uni-oldenburg.de", referrer: "https://elearning.uni-oldenburg.de/plugins.php/meetingplugin/index/config?cid=5ba3c47a8cb25d751f66ccfafec9efb6"
```

**Comment by Rillke** [comment]
This would be fixed by #231

**Comment by luniki** [comment]
Duplicate of #230
